src/main/java/net/mcreator/ui/modgui/TagGUI.java

```diff
@@ -176,7 +176,8 @@ public TagGUI(MCreator mcreator, ModElement modElement, boolean editingMode) {
 	@Override protected void afterGeneratableElementStored() {
 		super.afterGeneratableElementStored();
 		modElement.clearMetadata();
-		modElement.putMetadata("type", type.getSelectedItem().toString().toLowerCase());
+		modElement.putMetadata("type",

```

Comment:
Why?

Reply:
Tag's `type` metadata is used in `ElementUtil#getAllTags` which is added in this same PR if you looked at the PR's code

Reply:
> Tag's type metadata is used in ElementUtil#getAllTags which is added in this same PR if you looked at the PR's code

This was so even before your change, I can agree with that. However, this is not what you did, you added null check that does not make sense. type.getSelectedItem() can not return null as we always provide a selection of type.

Reply:
My commit's name is `Changes to avoid NPE` which means it's to avoid Null Pointer Exceptions, because my IDEA spotted there could have some